diff --git a/bootstrap b/bootstrap
index 56c68a7..ba910e7 100755
--- a/bootstrap
+++ b/bootstrap
@@ -209,7 +209,7 @@ fi
 #  not diagnosed by autoreconf itself (or could use some extra help
 #  explaining what to do in those cases).
 
-if grep AX_BOOST configure >/dev/null 2>&1; then
+if ! grep AX_BOOST configure >/dev/null 2>&1; then
     cat <<EOF
 It seems 'aclocal' could not find the autoconf macros for the Boost
 library.  These macros are available in the 'autoconf-archive'.  If
@@ -233,50 +233,6 @@ EOF
     exit 1
 fi
 
-#  Patch up AX_BOOST_BASE to suit our purposes
-#  While recent versions do support the ACTION-IF-[NOT-]FOUND API,
-#  they still produce a way too verbose message for the case where
-#  we can fall back to our included Boost sources.
-#  Older versions did not support the ACTION-IF-[NOT-]FOUND API.
-
-serial=`awk '/^#serial/{ if ($2>26) print $2 }' $macro_dir/ax_boost_base.m4`
-if test -n "$serial"; then
-    cat <<EOF >&2
-It seems there is a new AX_BOOST_BASE macro version out there.  The
-local patching that follows has not been tested on this version, so
-there may be trouble ahead.  If everything went fine for you please
-update the serial number used for this check in $0.
-
-When doing so, also check if our patch is still correct/necessary.
-EOF
-fi
-sed -i 's/\(-VERSION\]\))/\1, [ACTION-IF-FOUND], [ACTION-IF-NOT-FOUND])/' \
-    $macro_dir/ax_boost_base.m4
-sed -i '/"$succeeded" != "yes"/q' $macro_dir/ax_boost_base.m4
-sed -i '/"$succeeded" != "yes"/d' $macro_dir/ax_boost_base.m4
-cat <<EOF >> $macro_dir/ax_boost_base.m4
-    if test "\$succeeded" != "yes" ; then
-        if test "\$_version" = "0" ; then
-            AC_MSG_RESULT([no])
-        else
-            AC_MSG_RESULT([too old (\$_version)])
-        fi
-        # execute ACTION-IF-NOT-FOUND (if present):
-        ifelse([\$3], , :, [\$3])
-    else
-        AC_SUBST(BOOST_CPPFLAGS)
-        AC_SUBST(BOOST_LDFLAGS)
-        AC_DEFINE(HAVE_BOOST,,[define if the Boost library is available])
-        # execute ACTION-IF-FOUND (if present):
-        ifelse([\$2], , :, [\$2])
-    fi
-
-    CPPFLAGS="\$CPPFLAGS_SAVED"
-    LDFLAGS="\$LDFLAGS_SAVED"
-fi
-
-])
-EOF
 autoreconf                      # sync build infra-structure
 
 #  Make sure the top-level config.h and its dependencies can be found
