diff --git a/filters/doc-locate.cpp b/filters/doc-locate.cpp
index 4b09b29..4508274 100644
--- a/filters/doc-locate.cpp
+++ b/filters/doc-locate.cpp
@@ -260,17 +260,17 @@ public:
     clone.rotate (deskew_angle ());
 
     Magick::Geometry rv = bbox (clone);
-    if (HAVE_IMAGE_MAGICK_PP)
-    {
-      if (clone.page ().xNegative ())
-        rv.xOff (rv.xOff () - clone.page ().xOff());
-      else
-        rv.xOff (rv.xOff () + clone.page ().xOff());
-      if (clone.page ().yNegative ())
-        rv.yOff (rv.yOff () - clone.page ().yOff());
-      else
-        rv.yOff (rv.yOff () + clone.page ().yOff());
-    }
+    /*if (HAVE_IMAGE_MAGICK_PP)
+     *{
+     *  if (clone.page ().xNegative ())
+     *    rv.xOff (rv.xOff () - clone.page ().xOff());
+     *  else
+     *    rv.xOff (rv.xOff () + clone.page ().xOff());
+     *  if (clone.page ().yNegative ())
+     *    rv.yOff (rv.yOff () - clone.page ().yOff());
+     *  else
+     *    rv.yOff (rv.yOff () + clone.page ().yOff());
+    }*/
     return scale (rv);
   }
 
