diff --git a/connexions/usb.cpp b/connexions/usb.cpp
index 06dbeb8..a034585 100644
--- a/connexions/usb.cpp
+++ b/connexions/usb.cpp
@@ -99,7 +99,7 @@ using std::runtime_error;
             BOOST_THROW_EXCEPTION
               (runtime_error ("unable to initialise USB support"));
           }
-        libusb_set_debug (ctx_, 3);
+        libusb_set_option(ctx_, LIBUSB_OPTION_LOG_LEVEL, 3);
       }
 
     libusb_device **haystack;
